code/game/objects/items/weapons/bee_briefcase.dm

```diff
@@ -70,6 +74,6 @@
 		//Release up to 5 bees per use. Without using strange reagent, that means two uses. WITH strange reagent, you can get more if you don't release the last bee
 		for(var/bee = min(5, bees_left), bee > 0, bee--)
 			var/mob/living/simple_animal/hostile/poison/bees/syndi/B = new /mob/living/simple_animal/hostile/poison/bees/syndi(null)
-			B.master_and_friends = blood_list	//Doesn't automatically add the person who opens the case, so the bees will attack the user unless they gave their blood
+			B.master_and_friends = blood_list.Copy()	//Doesn't automatically add the person who opens the case, so the bees will attack the user unless they gave their blood
```

Comment:
Don't make circular list references like this @FalseIncarnate unless you specifically need to.

This causes oddities like here; you destroy the briefcase and, suddenly, all the bees that were previously your friends, turn hostile.